docs/guides/getting-started-orchestrate-microservices.md

```diff
@@ -23,23 +23,12 @@ While this guide uses code snippets in Java, you do not need to be a Java develo
 
 ## Prerequisites
 
-You must have access to either a local or remote Camunda Self-Managed installation or a SaaS account.
-
-<Tabs>
-   <TabItem value="sm" label="Self-Managed" default>
-      <details>
-         <summary>Have you installed Camunda yet?</summary>
-         <SmPrereqs/>
-         <Install/>
-      </details>
-   </TabItem>
-   <TabItem value="saas" label="SaaS">
-      <details>
-         <summary>Have you signed up for Camunda yet?</summary>
-         <SaasPrereqs/>
-      </details>
-   </TabItem>
-</Tabs>
+You must have access a Camunda 8 SaaS account.
```

Comment:
I think we mean "You must have access **to** a Camunda 8 SaaS account." here -- just a typo.